charts/pega/templates/_helpers.tpl

```diff
@@ -359,10 +359,22 @@ true
     sources:
     - secret:
         name: {{ template "pegaCredentialsSecret" $ }}
-  {{ if and (.Values.global.jdbc.external_secret_name) (not .Values.global.jdbc.password) }}
+  {{ if ((.Values.global.jdbc).external_secret_name) }}
```

Comment:
Can't it be written like this
`{{ if (.Values.global.jdbc.external_secret_name) }}.`
Is the sequence of parentheses correct?

Reply:
This is to handle null scenarios .
However .Values.global.jdbc is never expected to be null , added here as well to maintain consistency with other secrets